book/src/super-sql/functions/errors/quiet.md

```diff
@@ -1 +1,57 @@
-# quiet
+### Function
+
+&emsp; **quiet** &mdash; quiet "missing" errors
+
+### Synopsis
+
+```
+quiet(val: any) -> any
+```
+
+### Description
+
+The `quiet` function returns its argument `val` unless `val` is
+`error("missing")`, in which case it returns `error("quiet")`.
+Various operators and functions treat quiet errors differently than
+missing errors, in particular, dropping them instead of propagating them.
+Quiet errors are ignored by operators `aggregate`, `cut`, and `yield`.
+
+### Examples
+
+---
+
+_A quiet error in `values` produces no output_
+
+```mdtest-spq
+# spq
+values quiet(this)
+# input
+error("missing")
+# expected output
+```
```

Comment:
We might need some help from @nwt, as right now this is rendering blank. Based on problems we've had in the past, it's looking like some corner case related to the expected output being empty.

<img width="852" height="290" alt="image" src="https://github.com/user-attachments/assets/26b93eb7-593b-4a86-ad21-a601f00a14b2" />

Reply:
This is fixed now with @nwt's recent fix merged to main.